.github/workflows/update-master-from-upstream.yml

```diff
@@ -0,0 +1,20 @@
+name: Update master from upstreame
```

Comment:
spelling 'upstream'

Reply:
You are reviewing things in my fork... I fixed in a later commit.